scripts/setup_test.py

```diff
@@ -64,28 +72,35 @@ def setUp(self):
             'rename_is_called': False,
             'delete_file_is_called': False
         }
-        self.urls = []
-        def mock_create_directory(unused_path):
+        self.urls: List[str] = []
+        def mock_create_directory(unused_path: str) -> None:
```

Comment:
Please add empty lines above the function defs.

Reply:
Added.